packages/flutter/test/material/dropdown_test.dart

```diff
@@ -3165,4 +3166,77 @@ void main() {
         ..rrect(rrect: const RRect.fromLTRBXY(0.0, 0.0, 112.0, 47.0, 2.0, 2.0), color: Colors.grey[50], hasMaskFilter: false)
     );
   });
+
+  group('feedback', () {
+    late FeedbackTester feedback;
+
+    setUp(() {
+      feedback = FeedbackTester();
+    });
+
+    tearDown(() {
+      feedback.dispose();
+    });
+
+    Widget feedbackBoilerplate({bool? enableFeedback}) {
+      return MaterialApp(
+        home : Material(
+          child: DropdownButton<dynamic>(
```

Comment:
This test file seems to have been over-factored.  Maybe use `DropdownButton<String>` here and `onChanged(String? value) { }` for onChanged.

Reply:
Actually, if I change it to `DropdownButton<String>`, I can't write something like
`await tester.tap(find.byType(DropdownButton));` as this would not find the DropdownButton. Furthermore, I cannot use `await tester.tap(find.widgetWithText(InkWell, 'One'));` before tapping on the DropdownButton itself.